### PR TITLE
Testing/favorites

### PR DESCRIPTION
**Pull request by tfoxcollis**
## Summary:
-Break out favorites tests to their own file
-Break up home testing
-Add test for daily quote

## Type of Changes:
- [ ] Bug fix
- [x] Refactor
- [x] New feature

## How was this tested:
Cypress.io

### CODE REVIEW

**Comment by VictoriaFC** [comment]
Looks great! Thanks for refactoring the tests and adding additional tests!

**Review by abodnar1 — COMMENTED** [review]
I hadn't thought to do two intercepts in the `beforeEach()`! Looks good.

File: cypress/e2e/favorites_spec.cy.ts [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+    cy.contains('h2', 'Favorites')
+  })
+
+  it('should display a message if user has no quotes favorited', () => {
+    cy.get('.favorites-container').should('have.length', 0)
+    cy.contains('h3', 'No favorites yet!')
+  })
+})
+
+describe('Favorites page with quotes', () => {
```
Comment (CorCanavan) [inline cypress/e2e/favorites_spec.cy.ts:17]: Genius!

File: cypress/e2e/home_spec.cy.ts [hunk 2]
```diff
@@ -1,39 +1,56 @@
+/// <reference types="cypress" />
+
 describe('Homepage', () => {
   beforeEach(() => {
     cy.intercept('GET', 'https://zenquotes.io/api/quotes/11ef57ae8191dde524535934c158c4543950e06c&keyword=fairness', {fixture: "fairness_quotes"})
+    cy.intercept('GET', 'https://zenquotes.io/api/today?key=11ef57ae8191dde524535934c158c4543950e06c', {fixture: "daily_quote"})
```
Comment (CorCanavan) [inline cypress/e2e/home_spec.cy.ts:6]: Thank you for adding this in for the daily quote!

**Pull request merged**